File: 백준/Bronze/2562. 최댓값/최댓값.py
num = []  # 빈 리스트 num을 생성. 입력받은 숫자들이 저장될 예정
for i in range(9):  # for 반복문을 이용하여 9번 반복을 수행
    num.append(int(input()))  # append 함수를 이용해 끝에 하나씩 추가
print(max(num))  # 최댓값을 출력.
print(num.index(max(num)) + 1)  # index는 0부터 시작하므로 index값에 1을 더해서 몇 번째 수인지 출력.

File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        int[] arr = new int[9];
        int max = 0; // 최댓값 초기화
        int index = 0; // 자릿값 초기화

        for(int i=0; i<9; i++){ // 9번 반복하여
            arr[i] = sc.nextInt(); // 배열 값 입력 받기

            if(max < arr[i]) { // 입력받은 값이 max보다 클 경우
                max = arr[i]; // max에 값 저장
                index = i + 1; // 인덱스는 0부터 시작하므로 +1 
            }
        }
        sc.close();
        System.out.println(max);
        System.out.println(index);
    }
}